core/src/test/scala/unit/kafka/security/auth/ZkAuthorizationTest.scala

```diff
@@ -55,9 +56,11 @@ class ZkAuthorizationTest extends ZooKeeperTestHarness with Logging{
   @Test
   def testIsZkSecurityEnabled() {
     assertTrue(JaasUtils.isZkSecurityEnabled())
+    Configuration.setConfiguration(null)
```

Comment:
I'm wondering why you've added these two calls to `setConfiguration(null)`. I assume it is because the old issue we encountered a few times that we need to reset the configuration, but I can't remember seeing that happening here, have you?